src/features/genericContentPage/renderMethods.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { BLOCKS } from '@contentful/rich-text-types'
-import styled from 'styled-components'
 import { ReactTypeformEmbed } from 'react-typeform-embed'
 import Button from '../../components/button'
 import renderSponsors from '../../components/sponsors/helpers'
 import SponsorsSubsection from '../../components/sponsors/components/sponsorSubsection'
 import Figure from './components/figure'
 import Video from './components/video'
 
-const StyledButton = styled(Button)`
-  margin-bottom: 1.25em;
-`
-
 export const renderFigure = node => <Figure {...node.data.target.fields} />
 
 const renderStyledButton = ({ primary, to, content }) => (
-  <StyledButton primary={primary} to={to}>
+  <Button
+    variant={primary ? 'primary' : 'secondary'}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline between consequent and alternate of ternary expression.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)